intel/intel_decode.c

```diff
@@ -656,8 +656,9 @@ i915_get_instruction_dst(uint32_t *data, int i, char *dstname, int do_mask)
 
 	switch ((a0 >> 19) & 0x7) {
 	case 0:
-		if (dst_nr > 15)
-			fprintf(out, "bad destination reg R%d\n", dst_nr);
+                //The value of dst_nr must be between 0 and 15.
```

Comment:
Why is this commented out? 
I think this should either be removed completely, or log the error accordingly ?